pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -113,7 +192,177 @@ def load(fnames, tag=None, inst_id=None, file_cadence=dt.timedelta(days=1),
+        passed to `meta_processor` which should return a filtered version
+        of the input dict. The returned dict is loaded into a pysat.Meta
+        instance and returned as `meta`. (default=None)
+    meta_translation : dict or NoneType
+        Translation table used to map metadata labels in the file to
+        those used by the returned `meta`. Keys are labels from file
+        and values are labels in `meta`. Redundant file labels may be
+        mapped to a single pysat label. If None, will use
+        `default_from_netcdf_translation_table`. This feature
+        is maintained for compatibility. To disable all translation,
+        input an empty dict. (default=None)
+    drop_meta_labels : list or NoneType
+        List of variable metadata labels that should be dropped. Applied
+        to metadata as loaded from the file. (default=None)
+
+    Returns
+    -------
+    data : xarray.Dataset
+        Class holding file data
+    meta : pysat.Meta
+        Object containing metadata such as column names and units
+
+    Note
+    ----
+    This routine is intended to be used by pysat instrument modules supporting
+    a particular NASA CDAWeb dataset.
+
+    Examples
+    --------
+    ::
+
+        # within the new instrument module, at the top level define
+        # a new variable named load, and set it equal to this load method
+        # code below taken from cnofs_ivm.py.
+
+        # support load routine
+        # use the default CDAWeb method
+        load = functools.partial(cdw.load, pandas_format=False)
+
+    """
+
+    # Load data from any files provided
+    if len(fnames) <= 0:
+        return xr.Dataset()
+    else:
+        # Using cdflib wrapper to load the CDF and format data and
+        # metadata for pysat using some assumptions. Depending upon your needs
+        # the resulting pandas DataFrame may need modification
+        ldata = []
+        for lfname in fnames:
+            temp_data = cdflib.cdf_to_xarray(lfname, to_datetime=True)
+            ldata.append(temp_data)
+        # Combine individual files together
+        if len(ldata) > 0:
+            data = xr.combine_by_coords(ldata)
+
+    all_vars = io.xarray_all_vars(data)
+
+    # Convert output epoch name to 'time' for pysat consistency
+    if epoch_name != 'time':
+        if 'time' not in all_vars:
+            if epoch_name in data.dims:
+                data = data.rename({epoch_name: 'time'})
+            elif epoch_name in all_vars:
+                data = data.rename({epoch_name: 'time'})
+                wstr = ''.join(['Epoch label: "', epoch_name, '"',
+                                ' is not a dimension.'])
+                pysat.logger.warning(wstr)
+            else:
+                estr = ''.join(['Epoch label: "', epoch_name, '"',
+                                ' not found in loaded data, ',
+                                repr(all_vars)])
+                raise ValueError(estr)
+
```

Comment:
```suggestion

        epoch_name = 'time'

```

Reply:
This removes the warnings for me using 'roti'. The issue was the time metadata had min, maxes, etc as date times, and it was first in the dictionary of metadata. The pysat utilities filter this stuff from the epoch_name variable but in the ROTI case was coming in as `Epoch`.  Entirely possible the best form of the fix is changing something elsewhere, but forcing `epoch_name` to 'time' does sort it out for ROTI.